.gitignore

```diff
@@ -13,3 +13,4 @@ package-lock.json
 Desktop.ini
 ._*
 Thumbs.db
+.vscode/
```

Comment:
Stray addition?

Reply:
Well, I added it because someone might be using vscode to edit blogs. Same rationale as for .DS_Store and Desktop.ini